### DISCUSSION TITLE
Microsoft.OneDrive 64-bit or 32-bit? Installing with /allusers argument

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/winget-pkgs#171470] [Q&A] [answered]
Is the package for Microsoft OneDrive 64-bit or 32-bit?

Also, if I wanted to install it with the "/allusers" argument is this correct?

`winget install Microsoft.OneDrive --custom "/allusers"`

**Comment by SpecterShell** [comment, marked as answer]
The package contains both the 32-bit and the 64-bit installers, and supports both the user and the machine scopes.

To install the 64-bit one in machine scope, try `winget install Microsoft.OneDrive --architecture x64 --scope machine`

**Reply by myokeeh** [reply]
Thanks. Where is that documented? Is that "--scope machine" specific to OneDrive? How do custom arguments work for other installs? I know this might be getting out of topic.

**Reply by SpecterShell** [reply]
You can review the manifest content here: https://github.com/microsoft/winget-pkgs/blob/master/manifests/m/Microsoft/OneDrive/24.161.0811.0001/Microsoft.OneDrive.installer.yaml
